en/docs/basics/02_tensor.md

```diff
@@ -0,0 +1,185 @@
+print(x2)
+```
+
+
+Out:
+```text
+tensor([[1, 2],
+        [3, 4]], dtype=oneflow.int64)
+tensor([[1., 2.],
+        [3., 4.]], dtype=oneflow.float32)
+```
+
+We can see the tensor `x1` and `x2` are created, whose data types are INT64 and FLOAT32 respectively.
+
+### Create with NumPy array
+
+Tensors can be created from NumPy arrays, simply by passing the NumPy array as a parameter during the creation of the tensor object.
+
+```python
+x3 = flow.tensor(np.ones((2,3)))
+x4 = flow.tensor(np.random.rand(2,3))
+print(x3)
+print(x4)
+```
+Out:
+```text
+tensor([[1., 1., 1.],
+        [1., 1., 1.]], dtype=oneflow.float64)
+tensor([[0.6213, 0.6142, 0.1592],
+        [0.5539, 0.8453, 0.8576]], dtype=oneflow.float64)
+```
+
+
+### Created by an operator 
+
+There are also many operators available in OneFlow that you can use to create tensors. For example, [ones](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.ones#oneflow.ones), [zeros](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.zeros#oneflow.zeros) and [eye](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.eye#oneflow.eye) create all 1’s tensor, all 0’s tensor, and the unit tensor, respectively.
+
+
+```python
+x5 = flow.ones(2, 3)
+x6 = flow.zeros(2, 3)
+x7 = flow.eye(3)
+print(x5)
+print(x6)
+print(x7)
+```
+
+Out:
+```text
+tensor([[1., 1., 1.],
+        [1., 1., 1.]], dtype=oneflow.float32)
+tensor([[0., 0., 0.],
+        [0., 0., 0.]], dtype=oneflow.float32)
+tensor([[1., 0., 0.],
+        [0., 1., 0.],
+        [0., 0., 1.]], dtype=oneflow.float32)
+```
+
+The [randn](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.randn#oneflow.randn) method creates a randomized tensor:
+
+```python
+x8 = flow.randn(2,3)
+print(x8)
+```
+
+## The difference between `Tensor` and `tensor`
+There are two interfaces([oneflow.Tensor](https://oneflow.readthedocs.io/en/master/tensor.html?highlight=oneflow.Tensor#oneflow.Tensor) and [oneflow.tensor](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.Tensor#oneflow.tensor) ) in OneFlow, both of which can be used to create tensors. What’s the difference?
+
+
+Briefly speaking, the data type of `oneflow.Tensor` is limited to FLOAT32 by default, while the data type of `oneflow.tensor` can be changed when the data is created. The following code illustrates the difference:
+
+```python
+print(flow.Tensor([1, 2, 3]))
+print(flow.tensor([1, 2, 3]))
+print(flow.tensor([1.0, 2.0, 3.0]))
+```
+
+Out:
+```text
+tensor([1., 2., 3.], dtype=oneflow.float32)
+tensor([1, 2, 3], dtype=oneflow.int64)
+tensor([1., 2., 3.], dtype=oneflow.float32)
+```
+
+Besides, `oneflow.Tensor` can be created without specifying specific data:
+
+```python
+x9 = flow.Tensor(2, 3)
+print(x9.shape)
+```
+
+Out:
+```text
+flow.Size([2, 3])
+```
+
+Therefore, use `oneflow.Tensor` to create a tensor if you do not want to specify the data type, otherwise, you should use `oneflow.tensor`.
```

Comment:
```suggestion
Therefore, use `oneflow.Tensor` to create a tensor if you do not want to specify the data, otherwise, you should use `oneflow.tensor`.
```

原文的意思是，当不想指定具体数据时（不是数据类型），用大写 Tensor。我以上的建议修改应该也是还有问题的，想办法在优化下吧。

Reply:
已改为`specify an explicit value`